docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,251 @@ sidebar_label: Wikidata
 
 ## Overview
 
+OpenRefine includes powerful ways to both pull data from Wikidata and add data to it. 
 
+OpenRefine’s connections to Wikidata were formerly an optional extension, but are now installed automatically with the downloadable package. The Wikidata extension can be removed manually by navigating to your OpenRefine installation folder, and then looking inside `webapp/extensions/` and deleting the `Wikidata` folder inside. 
 
-*   Former extension, now core [but you can remove it].
-*   Account required
+You do not need a Wikidata account to reconcile your local OpenRefine project to Wikidata. If you wish to upload your cleaned dataset to Wikidata, you will need to register an account, and authorize OpenRefine with that account. 
 
+The best source for information about how OpenRefine works with Wikidata is [on Wikidata itself, under Tools](https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine). This has tutorials, guidelines on editing, and spaces for discussion and help.
 
-## Features
+## Reconciling with Wikidata
 
+Wikidata is, at this time, the most powerful [reconciliation service](reconciling) for OpenRefine. [It supports](https://reconciliation-api.github.io/testbench/):
```

Comment:
To avoid us having to update this (and have any discussions about what "most powerful" means in this context), can we just state the functions it supports?